src/functionalTest/resources/features/F-204 - Get NoC Questions/F-204.feature

```diff
@@ -144,6 +144,7 @@ Feature: F-204 Get Notice of Change Questions
       And a successful call [to update the applicant org policy] as in [204_Update_Org_Policies_With_Missing_Policy],
       And [The challenge questions have answers that resolve to three distinct case roles (1,2,3)] in the context of the scenario
       And [The configuration of the case creation event only establishes organisation policies containing case roles 1 and 2] in the context of the scenario
+      And a wait time of [15] seconds [to allow for the cache to refresh],
```

Comment:
why wait..? i thought with caseType version you never get stale data.  In any case 15s a lot which increases build (as we have so many NoC scenarios) ?

Reply:
The BEFTA tests are having issues with the cached data - 19 failures in total. Only these 19 tests have the wait period.